addon/templates/components/flexberry-tab-bar.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="ui tabular menu">
```

Comment:
Component already has its wrapping div, so this additional block inside is unnecessary, you should add this class names in components code through `classNames` array.